RR.App/Program.cs

```diff
@@ -61,8 +61,14 @@ public static void Main(params string[] args)
             });
 
             var connectionString = Environment.GetEnvironmentVariable("ConnectionStrings__Default");
-            builder.Services.AddDbContext<DatabaseContext>(options => options.UseNpgsql(connectionString));
-            builder.Services.AddDbContext<DatabaseContext>(options => options.UseNpgsql(configuration["ConnectionStrings:Default"]));
+            builder.Services.AddDbContext<DatabaseContext>(options => options.UseNpgsql(connectionString), ServiceLifetime.Transient);
+            builder.Services.AddTransient<DatabaseContext>(serviceProvider =>
+            {
+                var defaultConnectionString = serviceProvider.GetRequiredService<IConfiguration>()["ConnectionStrings:Default"];
```

Comment:
Please get the connection string from the environment variables

Reply:
Isn't this what line 64 does? Just making sure.